planemo/cwl/run.py

```diff
@@ -28,29 +38,34 @@
 class CwlToolRunResponse(SuccessfulRunResponse):
     """Describe the resut of a cwltool invocation."""
 
-    def __init__(self, runnable, log, outputs=None):
-        self._runnable = runnable
+    def __init__(
+        self,
+        log: str,
+        outputs: Optional[Dict[str, Union[Dict[str, Union[str, int]], int]]] = None,
```

Comment:
I wrote this code and don't understand that outputs type declaration - (Like why does that dictionary only container integers and strings) - my intuition says this should be ```Optional[Dict[str, Any]]``` instead. Or at least an ``Any`` on that inner ``Union``.

I'm happy to merge as is though and wait for a problem with this.

Reply:
@jmchilton Agreed, sometimes MonkeyType tries too hard! Fixed in a rebase.